src/libs/functions/formatRedirectLink.ts

```diff
@@ -4,22 +4,26 @@ import CozyClient, {
 } from 'cozy-client'
 
 export const formatRedirectLink = (
-  onboardingRedirection: string,
+  redirectLink: string,
   client: CozyClient
-): string => {
+): string | null => {
   const cozyUrl = client.getStackClient().uri
   const subDomainType = client.getInstanceOptions().capabilities.flat_subdomains
     ? 'flat'
     : 'nested'
 
-  const { slug, path, hash } = deconstructRedirectLink(onboardingRedirection)
+  try {
+    const { slug, pathname, hash } = deconstructRedirectLink(redirectLink)
 
-  return generateWebLink({
-    cozyUrl,
-    subDomainType,
-    slug,
-    pathname: path,
-    hash,
-    searchParams: []
-  })
+    return generateWebLink({
+      cozyUrl,
+      subDomainType,
+      slug,
+      pathname,
+      hash,
+      searchParams: []
+    })
+  } catch {
+    return null
```

Comment:
Would it be possible to `log.error()` here? So it would be sent to Sentry

Note that we use `getErrorMessage` in this project to ensure we send correct error message to sentry

Example: https://github.com/cozy/cozy-flagship-app/blob/d62f13e7377c40db68c4219be2657c0ff389ac87/src/hooks/useAppBootstrap.functions.ts#L77-L83

Reply:
Sure!